src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/VisualBasicCodeFixVerifier`2+Test.cs

```diff
@@ -28,7 +27,7 @@ public Test()
 
                 SolutionTransforms.Add((solution, projectId) =>
                 {
-                    var parseOptions = (VisualBasicParseOptions)solution.GetRequiredProject(projectId).ParseOptions!;
+                    var parseOptions = (VisualBasicParseOptions)solution.GetProject(projectId)!.ParseOptions!;
```

Comment:
Can we restore GetRequiredProject?

Reply:
There is no longer an IVT from this file linked into CodeStyle layer into Workspaces. Given there are only couple of uses of this API in our code fix verifiers, I felt it was fine to not introduce an IVT and just behave like a third party test framework helper.